common/wuanalysis/anawu.cpp

```diff
@@ -247,7 +247,7 @@ void WuScope::getAttr(StringBuffer & result, WuAttr attr) const
 {
     StringBuffer name;
     name.append('@').append(queryWuAttributeName(attr));
-    attrs->getProp(result, name);
+    result.set(attrs->queryProp(name));
```

Comment:
out of interest, how does this change the behaviour?

Reply:
getProp isn't returning anything (empty string). Could there be aproblem with getProp?  Eventually, I modified to use queryProp and it is working.

Reply:
getProp isn't returning any value...I've created a jira for this: https://track.hpccsystems.com/browse/HPCC-23072.  I'll investigate later.

Reply:
Jake has found the problem. I had getProp parameters the wrong way round

Reply:
That's counterintuitive!  I would have a made the same mistake.